packages/pages-shared/__tests__/metadata-generator/parseRedirects.valid.test.ts

```diff
@@ -45,6 +45,30 @@ test("parseRedirects should ignore comments", () => {
 	});
 });
 
+test("parseRedirects should ignore fragments and search on source", () => {
```

Comment:
And these are here too: https://github.com/cloudflare/workers-sdk/blob/main/packages/pages-shared/__tests__/metadata-generator/parseRedirects.valid.test.ts#L63

Reply:
I guess the fragment on source is missing.